content/aips/942c023868ca-AIP-STKAAVE_RETRIEVAL.md

```diff
@@ -0,0 +1,77 @@
+Created: 2022-10-10
+
+---
+
+# Simple Summary
+
+This proposal enables Balancer DAO to retrieve the stkAAVE Liquidity Mining incentives distributed to Linear Pools in the Balancer Boosted Aave Pool.
+
+# Abstract
+
+Balancer DAO seeks to retrieve around 1,500 stkAAVE from several Linear Pools. Balancer DAO accrued this stkAAVE after it deposited liquidity into Aave while Liquidity Mining incentives were being distributed, [1,2]. 
+
+The proposal is to implement the changes required to enable Balancer DAO to retrieve the stkaAAVE from the Linear Pools within the Balancer Boosted Aave Pool.
+
+# Motivation
+
+Three Linear Pools within the Balancer Boosted Aave Pool have accrued around 1,500 stkAAVE from Liquidity Mining rewards, [3]. 
+
+The Linear Pool holds the aTokens and receives the stkAAVE liquidity mining rewards but can not call the `claimRewards` function or manage any stkAAVE if it was received.
+
+The Linear Pools unable to call `getUnclaimedRewards` with [`0xba12222222228d8ba445958a75a0704d566bf2c8`](https://etherscan.io/address/0xba12222222228d8ba445958a75a0704d566bf2c8) are detailed below:
+
+[0x02d60b84491589974263d922d9cc7a3152618ef6](https://etherscan.io/address/0x02d60b84491589974263d922d9cc7a3152618ef6)
+[0xd093fa4fb80d09bb30817fdcd442d4d02ed3e5de](https://etherscan.io/address/0xd093fa4fb80d09bb30817fdcd442d4d02ed3e5de)
+[0xf8fd466f12e236f4c96f7cce6c79eadb819abf58](https://etherscan.io/address/0xf8fd466f12e236f4c96f7cce6c79eadb819abf58)
+
+Aave enables other addresses to claim on behalf of smart contracts using the `claimOnBehalfOf` function. 
+
+In order to do this, a specified address must be declared the valid owner of the contract.
+
+# Specification
+
+Set Balancer contract [`0x10a19e7ee7d7f8a52822f6817de8ea18204f2e4f`](https://etherscan.io/address/0x10a19e7ee7d7f8a52822f6817de8ea18204f2e4f) as the StkAaveRetrieval contract in the incentives controller contract, [4].
+
+Snapshot and forum discussion can be found at reference [6] and [7].
+
+# Test Cases
+
+This proposal has been tested and peer reviewed by Bored Ghost Developing, [5].
+
+Test cases, payload, and retrieval contract can be found at reference [8], [9] & [10] respectively.
+
+# Implementation
+ 
+Call setClaimer(0xba12222222228d8ba445958a75a0704d566bf2c8, RETRIEVAL_CONTRACT_ADDR) for Balancer contracts on INCENTIVES_CONTROLLER_PROXY_ADDRESS
+ 
+Balancer DAI-aDAI Linear Pool [0x02d60b84491589974263d922d9cc7a3152618ef6](https://etherscan.io/address/0x02d60b84491589974263d922d9cc7a3152618ef6)
+
+Balancer USDC-aUSDC Linear Pool [0xd093fa4fb80d09bb30817fdcd442d4d02ed3e5de](https://etherscan.io/address/0xd093fa4fb80d09bb30817fdcd442d4d02ed3e5de)
+
+Balancer USDT-aUSDT Linear Pool
+[0xf8fd466f12e236f4c96f7cce6c79eadb819abf58](https://etherscan.io/address/0xf8fd466f12e236f4c96f7cce6c79eadb819abf58)
+ 
+Incentives controller proxy [0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5[(https://etherscan.io/address/0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5#readProxyContract) 
```

Comment:
Make the Implementation more succinct and clear. Use numbered lists to specify the steps (if any) more clearly. 

Take a look at https://raw.githubusercontent.com/aave/aip/Pending-AIPs/content/aips/4A7DEC60B007-AIP-AAVE-LLAMA.md on how to write the Specification and Implementation sections.